router/dataplane.go

```diff
@@ -3631,6 +4035,7 @@ func (p *scionPacketProcessor) process(
 		// @ p.d.InDomainExternalInForwardingMetrics(egressID)
 		// @ assert absPkt(ub) == AbsProcessEgress(nextPkt)
 		// @ nextPkt = absPkt(ub)
+		// @ assert slayers.IsSupportedPkt(ub) == old(slayers.IsSupportedPkt(ub))
```

Comment:
is this assertion needed? It feels like it was originally here for debugging purposes but I'm not sure if it is worth keeping